src/img/container-delete.svg

```diff
@@ -3,7 +3,7 @@
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 12 12" style="enable-background:new 0 0 12 12;" xml:space="preserve">
 <style type="text/css">
-	.st0{fill:#858585;}
+	.st0{fill:#FFFFFF;}
```

Comment:
Maybe show me how to do this tomorrow?

Reply:
In case this actually is what's blocking the PR:

Replace 
```html
<path class="st0" ...
```
with
```html
<path fill="#FFFFFF" ...
```

and you can then remove the entire `<style />` block. 

You _might_ also want to run it through e.g. svgo (via [svgomg](https://jakearchibald.github.io/svgomg/) for easy use).

(Thanks for the work behind this PR, it's a great improvement and I'm forward to seeing it merged!)